localpublish.props

```diff
@@ -0,0 +1,9 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Sorry for this, had to do lots of tactical changes to get our build off the floor.

-Eric




On Thu, Oct 22, 2015 at 8:44 AM -0700, "Jose Perez Rodriguez" <notifications@github.com<mailto:notifications@github.com>> wrote:


In localpublish.props<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2fbuildtools%2fpull%2f315%23discussion_r42765425&data=01%7c01%7cEric.St.John%40microsoft.com%7cb20f64c8445641cc981408d2daf7a2c7%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=4z6xKx7o4%2fx4s%2fyFZtswx3RXCW47OBW4I0YkwRJpTqo%3d>:

> @@ -0,0 +1,9 @@
> +<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">


NVM, I saw you removed it in a later commit. Thanks!

-
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2fbuildtools%2fpull%2f315%2ffiles%23r42765425&data=01%7c01%7cEric.St.John%40microsoft.com%7cb20f64c8445641cc981408d2daf7a2c7%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=llXpQvJ3QeJJDc%2buAIc%2bc3E62HLZRbbYbojQIa2g78U%3d>.